pyproject.toml

```diff
@@ -79,6 +79,7 @@ pur = "^5.4.2"
 Sphinx = "^4.3.2"
 sphinx-rtd-theme = "^1"
 twine = "^3"
+isort = "^5.10.1"
```

Comment:
This should not be needed as the pre-commit hook should provide the dependency. We also don't have poetry dependencies for black, flake8 but pre-commit hooks for them.

Reply:
Okay. Give me just a few minutes to remove it

Reply:
Though doesn't it make sense to include them as dev dependencies? They are in fact dev dependencies and could be used before the hooks.